src/condor_tests/test_job_submit_method_recording.py

```diff
@@ -7,21 +7,35 @@
 #them for evaluation.
 #
 #Written by: Cole Bollig @ March 2022
+#
+#Tickets:
+# Base: HTCONDOR-996
```

Comment:
Generally speaking, we don't need stuff like this in the source tree, as git has all this information, and it is the authoritative source of truth.